ngraph/frontend/onnx_import/CMakeLists.txt

```diff
@@ -66,6 +68,9 @@ target_include_directories(onnx_importer SYSTEM PUBLIC $<BUILD_INTERFACE:${ONNX_
 target_include_directories(onnx_importer SYSTEM PRIVATE ${NGRAPH_INCLUDE_PATH}
         ${ONNX_INCLUDE_DIR} ${ONNX_PROTO_INCLUDE_DIR} ${Protobuf_INCLUDE_DIRS})
 
+target_include_directories(onnx_importer PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src
+                                                 ${CMAKE_CURRENT_SOURCE_DIR}/include/onnx_import)
```

Comment:
I suppose we don't need `${CMAKE_CURRENT_SOURCE_DIR}/include/onnx_import` since `ONNX_IMPORT_INCLUDE_DIR` is already included

Reply:
My Idea was to remove onnx_import prefix from private files.
for example:
`#include "onnx_import/core/node.hpp" -> #include "core/node.hpp"`

Reply:
Why do we need it? `onnx_import` prefix denotes that public header is used

Reply:
@tsocha I agree with Ilya. In the nGraph core we have the same situation but we use the `<ngraph/` prefix for all public headers.

Reply:
The point is - we should include public headers in the same way as library users do it

Reply:
Ok I will do that.